### PR TITLE
77729: summary view

### PR DESCRIPTION
**Pull request by bratli**

### CODE REVIEW

**Comment by bratli** [comment]
@paal123 @arildbjerkenes  okay to just merge?

File: src/modules/InvitationForPunchOut/views/CreateIPO/Summary/Summary.tsx [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+                </Table>
+            </TableSection>
+            <TableSection>
+                <Typography variant="h5">Selected scope</Typography>
+                <Table>
+                    <Head>
+                        { getHeaders() }
+                    </Head>
+                    <Body>
+                        { mcScope.length > 0 && mcScope.map(mcPkg => getMcScope(mcPkg)) }
```
Comment (paal123) [inline src/modules/InvitationForPunchOut/views/CreateIPO/Summary/Summary.tsx:151]: should both be either getXXPkgScope or getXXScope?

**Review by paal123 — APPROVED** [review]
With suggestions

File: src/modules/InvitationForPunchOut/views/CreateIPO/Summary/Summary.tsx (hunk 1 above)
Comment (bratli) [reply src/modules/InvitationForPunchOut/views/CreateIPO/Summary/Summary.tsx:151]: Will fix this in next case since I have branched out from here :)

**Pull request merged**